src/backend/access/external/pxffilters.c

```diff
@@ -125,7 +126,48 @@ dbop_pxfop_map pxf_supported_opr[] =
 	{1871 /* int82gt */, PXFOP_GT},
 	{1872 /* int82le */, PXFOP_LE},
 	{1873 /* int82ge */, PXFOP_GE},
-	{1869 /* int82ne */, PXFOP_NE}
+	{1869 /* int82ne */, PXFOP_NE},
+
+	/**************FLOAT****************/
+	/* float4 */
+	{Float4EqualOperator  /* float4eq */, PXFOP_EQ},
+	{622  /* float4lt */, PXFOP_LT},
+	{623 /* float4gt */, PXFOP_GT},
+	{624 /* float4le */, PXFOP_LE},
+	{625 /* float4ge */, PXFOP_GE},
+	{621 /* float4ne */, PXFOP_NE},
+
+	/* float8 */
+	{Float8EqualOperator  /* float8eq */, PXFOP_EQ},
+	{672  /* float8lt */, PXFOP_LT},
+	{674 /* float8gt */, PXFOP_GT},
+	{673 /* float8le */, PXFOP_LE},
+	{675 /* float8ge */, PXFOP_GE},
+	{671 /* float8ne */, PXFOP_NE},
+
+	/* float48 */
```

Comment:
This pull request doesn't quite seem related to supporting operators on Float. What is the purpose of this ?